Lib9c/CurrencyExtensions.cs

```diff
@@ -26,8 +41,27 @@ public static Currency Deserialize(Bencodex.Types.Dictionary serialized)
             {
                 minters = mintersAsList.Select(b => new Address(((Binary) b).ByteArray)).ToImmutableHashSet();
             }
-            
-            return new Currency((Text)serialized["ticker"], ((Binary)serialized["decimalPlaces"]).First(), minters);
+
+            if (serialized.ContainsKey("totalSupplyTrackable"))
+            {
+                if (serialized.ContainsKey("maximumSupplyMajor"))
+                {
+                    return Currency.Capped(
```

Comment:
Is there any case of `Currency.Uncapped()`?

Reply:
In case the total supply is trackable and maximum supply cap is present, Currency.Capped() is used. If total supply is trackable and maximum supply cap is not present, Currency.Uncapped() is used (as in line 59 of this same file.)

Reply:
Oh.. I understand. Thanks!